# module_8_2.py
"Сложные моменты и исключения в стеке вызовов функции".
